frontend/src/pages/login/Login.tsx

```diff
@@ -0,0 +1,169 @@
+                            Sign in to your account
+                        </h2>
+                        <p className="mt-2 text-center text-sm text-gray-600">
+                            Or Sign up for a new account as a {" "}
+                            <Link
+                                to="/sign-up/student"
+                                className="font-medium text-[#ca3448] hover:text-[#E9D3D7]"
+                            >
+                                Student
+                            </Link>{" "}
+                            or{" "}
+                            <Link
+                                to="/sign-up/teacher"
+                                className="font-medium text-[#ca3448] hover:text-[#E9D3D7]"
+                            >
+                                Teacher
+                            </Link>
+                        </p>
+                    </div>
+                    <form className="mt-8 space-y-6" action="#" method="POST">
+                        <input type="hidden" name="remember" defaultValue="true" />
+                        <div className="-space-y-px rounded-md shadow-sm">
+                            <div>
+                                <label htmlFor="email" className="sr-only">
+                                    Email address
+                                </label>
+                                <input
+                                    id="email"
+                                    name="email"
+                                    type="email"
+                                    required
+                                    className="relative block w-full appearance-none rounded-t-md border border-gray-300 px-3 py-2 placeholder:text-gray-400 focus:border-[#ca3448] focus:outline-none focus:ring-[#ca3448] sm:text-sm"
+                                    placeholder="Email address"
+                                />
+                            </div>
+                            <div>
+                                <label htmlFor="password" className="sr-only">
+                                    Password
+                                </label>
+                                <input
+                                    id="password"
+                                    name="password"
+                                    type="password"
+                                    required
+                                    className="relative block w-full appearance-none rounded-b-md border border-gray-300 px-3 py-2 placeholder:text-gray-400 focus:border-[#ca3448] focus:outline-none focus:ring-[#ca3448] sm:text-sm"
+                                    placeholder="Password"
+                                />
+                            </div>
+                        </div>
+                        <div className="flex items-center justify-between">
+                            <div className="flex items-center">
+                                <input
```

Comment:
So I guess for the input `remember-me` if it's checked, the user will have an infinite login session?

Also, don't forget to add the `value="true"` or else the backend will not be receiving if the user checked it or not

Reply:
Yes, infinite session but it also have to deal with session token in a cookie. I think for now we can focus on the login authorization and link the db if we have time we might work on it if anyone wants.

Reply:
value="true" added the value is true now!